modules/help.sh

```diff
@@ -83,6 +83,7 @@ services                                  executes the given command on the spec
    [musicblocks]                          MusicBlocks is a programming language for exploring musical concepts in an fun way
    [minetest]                             Minetest is an open source infinite-world block sandbox game engine with survival and crafting
    [invoiceninja]                         Invoiceninja is the leading self-host platform to create invoices.
+   [grocy]                                grocy is web-based, self-hosted groceries and household management utility for your home
```

Comment:
Need to make sure that the 3 lines in `README.md`, `help.sh`, and `services.sh` are the same.